x-pack/plugin/eql/src/test/resources/mapping-default.json

```diff
@@ -56,7 +56,7 @@
             "type" : "text",
             "fields" : {
                 "keyword" : {
-                    "type" : "keyword",
+                    "type" : "wildcard",
```

Comment:
I thought we can't tell whether something is a wildcard or keyword.

Reply:
We cannot, but this doesn't mean we shouldn't test different field types.